recipe/meta.yaml

```diff
@@ -27,6 +27,7 @@ requirements:
     - python
     - setuptools
     - libxml2
+    - {{ cdt('numactl-devel') }}  # [ppc64le]
```

Comment:
CDTs should be in `build`, see https://conda-forge.org/docs/maintainer/knowledge_base.html#core-dependency-tree-packages-cdts

Reply:
Fixed this change in the latest commit